src/vm/methodtablebuilder.cpp

```diff
@@ -11024,11 +11078,13 @@ void MethodTableBuilder::VerifyVirtualMethodsImplemented(MethodTable::MethodData
             MethodTable::MethodIterator it(hData);
             for (; it.IsValid() && it.IsVirtual(); it.Next())
             {
-                if (it.GetTarget().IsNull())
-                {
-                    MethodDesc *pMD = it.GetDeclMethodDesc();
-                    BuildMethodTableThrowException(IDS_CLASSLOAD_NOTIMPLEMENTED, pMD->GetNameOnNonArrayClass());
-                }
+                // @DESIGN - What is the right level of check if the interface itself does not have default implementation
```

Comment:
Please add a DIM_TODO to the comment so we can find these bits of commented out logic.